src/Amazon.Lambda.Tools/Amazon.Lambda.Tools.csproj

```diff
@@ -15,7 +15,7 @@
     <Copyright>Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.</Copyright>
     <Product>AWS Lambda Tools for .NET CLI</Product>
     <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    <Version>5.11.0</Version>
```

Comment:
you shouldn't be manually updating the version. This will be done part of the release process. Also, please add a change file to the PR https://github.com/aws/aws-extensions-for-dotnet-cli/blob/master/CONTRIBUTING.md#adding-a-change-file-to-your-contribution-branch

Reply:
Done